### DISCUSSION TITLE
Reduce the amount of captchas in the game

### DISCUSSION THREAD

**Discussion by Xeko-92** [sunflower-land/sunflower-land#3543] [Ideas]
Since I don't know where else to turn, I'm posting this here. We need to get rid of captchas. I've been talking about this for a while now in discord, but whenever I bring it up, I get dismissed by yes-men or what I'm posting gets deleted by mods. At best, captchas slow bots down, at worst, they do nothing other than make the UX for actual player much worse. We needed captchas to combat bots, okay sure, but that should've only been a temporary measure, they don't work. 

I've been playing SFL since late 2021, and to be honest, if I hadn't been playing this long, I wouldn't be playing right now. If I found this game tomorrow and realized I had to solve captchas all day, I would quit immediately. This isn't late 2021 anymore, there are a ton of actual decent crypto/NFT games out now, people won't play a game just because it exists like when I started this. It needs to be an actual fun game because peoples' time is limited, and captchas aren't fun.

This time stamped video is what was deleted by the mods: https://www.youtube.com/watch?v=e_g-xxbpqDY&t=1808s

![post deleted](https://github.com/sunflower-land/sunflower-land/assets/123136614/cd59b3de-765e-4dcd-81c0-8d691eb2a2ad)

**Comment by Dionis404** [comment]
What suggestions would you have?  You're just complaining that there is a captcha that allows you to fight bots that speed up halving and kill the game.

**Reply by Xeko-92** [reply]
There's no good way to stop bots. If there was, every game would've implemented it by now. I just know that 1) captchas don't work and 2) no good games have captchas. Just removing captchas without adding anything else would be better than the game currently is.

Pixels has a reputation system and I'm sure other things behind the scenes. Maybe try something like that? I'm not a dev, I just know that the current system isn't working. You could argue that the game is "fine", but how many DAUs do we have? Are we actually growing during this bull run? There is way too much clicking required right now, especially when needing to farm sunflowers or potatoes.

**Reply by Dionis404** [reply]
Lol there are a lot of bots in Pixels that ruin the game. That's one of the reasons I stopped playing it.  I suggest you check out the plans for the upcoming updates to see if we're growing or not.

**Reply by Xeko-92** [reply]
I didn't say there weren't bots in Pixels, I just said what they're doing. I guarantee there are bots in SFL too, it's just not as obvious because they stay on their farms.

**Reply by eliasSFL** [reply]
Hey guys, I suggest u wait out for the announcement on Tuesday on discord and twitter. There are big changes coming that can affect the $SFL economy

**Reply by Dionis404** [reply]
Yeah I know there's gonna be some big changes in the game, I'm looking forward to Tuesday.

**Comment by Xeko-92** [comment]
>I suggest you check out the plans for the upcoming updates to see if we're growing or not.

The daily chest was added so that the devs could see DAUs, and according to dapp radar, our DAUs have not increased at all recently, in fact, we're actually down from the high around a year ago. So I don't think we are growing.

**Comment by Xeko-92** [comment]
If they want to keep captchas in, at least adjust them for players who've been playing a long time. When I'm harvesting 50 crops, why do I need to solve 5 captchas? It's ridiculous. At most I should be doing like 1 or 2 captchas a day.

**Comment by Xeko-92** [comment]
> Hey guys, I suggest u wait out for the announcement on Tuesday on discord and twitter. There are big changes coming that can affect the $SFL economy

I didnt hear anything about the changes affecting captchas.

**Comment by Xeko-92** [comment]
I complained about this on discord again today. A couple ideas were brought up about how to fix it:

Reduce captchas for upgrading islands
A skill for high level players or reducing it once your bumpkin hits a certain level.

We just need something. It feels like we've had captchas forever and it's never going to be something that's actually fixed/worked on. It should've been a temporary measure to combat bots and it just feels like it's now the worst part of the game that's never going to get better.

**Comment by MrSnorch** [comment]
I fully support the idea of reducing the number of captchas per day 👍

**Comment by Jericho-rpc** [comment]
Yes, you are right. These verifications will only affect the game experience and hinder players. They will not prevent scripts. Note that it has been 2 years since the game started. They do not have any countermeasures to solve script bots. Instead, It has been hindering normal players from enjoying the game. The project side acted slowly and lazily, and the old players of the game were constantly losing and decreasing. They did not realize what the problem was, but instead suppressed the voices of those who raised objections.

**Comment by Xeko-92** [comment]
Any word from any devs if we ever might move past Captcha Land into a decent spot?

**Comment by Jericho-rpc** [comment]
You can see that the project team is lazy and arrogant, they don't care about the player's opinion voices, they will only put their own interest development first, that's why this game is a few years old the land has been growing but the DAU is still dropping consistently, have they done anything for the slow progress of the project? It seems to continue to spin in place

**Comment by Xeko-92** [comment]
With the recent announcement in discord regarding all the bots, I'm bumping this thread.

At the very least, we need to reduce the amount of captchas in the game, it doesn't seem to be serving much of a purpose at this point.

**Comment by Xeko-92** [comment]
With this being a week where minor improvements are being made, now would be a good time to reduce the amount of captchas in the game

**Comment by Alucard0389** [comment]
Maybe people how have the life time banner should get significant less captcha.

**Comment by Xeko-92** [comment]
![image](https://github.com/user-attachments/assets/4e9eef1d-9291-4717-aea9-c045f26731a5)

**Comment by Jericho-rpc** [comment]
The devs won't notice, and even if they see it, they won't act on it they claim to be a small team acting quickly, they say they listen to the players and in reality do nothing, captcha brings no resistance to the bots and hinders the normal players instead
